pkg/capr/planner/etcdrestore.go

```diff
@@ -49,24 +51,28 @@ if [ -z "$KUBECONFIG" ]; then
         exit 1
 fi
 
-TMPMACHINEIDS=$(mktemp)
+MACHINEIDSFILE="$1"
+NODENAMESFILE="$2"
+
+if [ -z "$MACHINEIDSFILE" ] || [ -z "$NODENAMESFILE" ]; then
+        echo "Must define nodenames file and machineids file"
```

Comment:
Question: Why not exit here? When neither MACHINEIDSFILE nor NODENAMESFILE is set, it will result in deleting all nodes.